recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
-{% set version = "1.6.2.3" %}
+{% set version = "1.7.0.1" %}
 
 package:
   name: cutensor
   version: {{ version }}
 
 source:
-  url: https://developer.download.nvidia.com/compute/cutensor/redist/libcutensor/linux-x86_64/libcutensor-linux-x86_64-{{ version }}-archive.tar.xz  # [linux64]
```

Comment:
Asked internally, this is a one-off change that we should revert in the future. This was also why the CF bot could not send the update to us.

Reply:
The url is fixed, need to revert the change.